templates/masterclasses.html

```diff
@@ -52,17 +51,12 @@ <h3 class="p-media-object__title">
                   <li class="p-media-object__meta-list-item--date">
                     <span class="u-off-screen">Date: </span>{{ session.Date.Formatted }}
                   </li>
-                  <!-- Add to calendar button, URL documentation: https://github.com/InteractionDesignFoundation/add-event-to-calendar-docs/blob/main/services/google.md -->
-                  <!-- Currently, all events start at 11AM CST with duration of 45 minutes -->
+                  <!-- Add to calendar btn uses a gcal published event https://support.google.com/calendar/answer/41207 -->
                   <li class="p-media-object__meta-list-item--date">
-                    <a href="https://calendar.google.com/calendar/render?action=TEMPLATE&text=Masterclass:+{{ session.Topic }}&dates={{ session.Date.Calendar }}T110000/{{ session.Date.Calendar }}T114500&ctz=Europe/Paris&location={{ session.Event }}&details={{ session.Owner }}+would+like+to+talk+about+{{ session.Topic }}.%0ANotes:+{{ session.Notes }}"
-                      target="_blank" rel="noopener noreferrer">
+                    <a href="{{ session.Event }}" target="_blank" rel="noopener noreferrer">
```

Comment:
I am a bit confused, the event column in the google doc changed to be using a template that creates the single event. Maybe add some doc about how to generate this link for the future -- maybe in the spec.

But on the other hand: Didn't we want a shared calendar for all events, that users can subscribe to? In that case we probably don't want a add to calendar link per session.

Reply:
I'll document what the column corresponds to in the website.

And yes, I'm working on making a "Subscribe to Masterclasses" button, that deprecates the "Add to calendar" button. The subscribe button will contain [this](https://calendar.google.com/calendar/u/0?cid=Y19iNjdjNzE1M2MzMzA4ZDM5Y2RjNzg2YjBjOWMzMDExNWY3MDg2YTc0NjQ0ZTYxYTFlZDFkYmJiYTdlOTc3YzAwQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20) link to the shared calendar. 

Non canonical users would not be able to see event details, only a "busy" event:
<img width="895" alt="image" src="https://github.com/canonical/masterclasses.canonical.com/assets/54525904/e6c35fe3-8a49-40b6-83fe-e2cc8737357d">